README.md

```diff
@@ -218,40 +218,64 @@ Configuration of Tensorflow for a model is done through the Parameters section o
 * `TF_GRAPH_TAG`: Tag of the graphs to use. See [protobuf here](https://github.com/tensorflow/tensorflow/blob/6f72753a66d6abab8b839cc263a9f1329861f6f9/tensorflow/core/protobuf/meta_graph.proto#L56)
 * `TF_SIGNATURE_DEF`: Signature def to use. See [protobuf here](https://github.com/tensorflow/tensorflow/blob/6f72753a66d6abab8b839cc263a9f1329861f6f9/tensorflow/core/protobuf/meta_graph.proto#L260-L331)
 * `MAX_SESSION_SHARE_COUNT`: This parameter specifies the maximum number of [model instances](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#instance-groups) that can share a [TF session](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/public/session.h). The default value is 1 which means Triton will create a separate TF session for each model instance. If this parameter is set to the total number of instances, then Triton will create only a single TF session which will be shared by all the instances. Sharing TF sessions among model instances can reduce memory footprint of loading and executing the model.
+* `TF_INIT_OPS_FILE`: This parameter specifies the name of the file in JSON
+format that contains the [initialization operations](https://www.tensorflow.org/api_docs/python/tf/compat/v1/global_variables_initializer).
+The JSON file must have a single element named 'init_ops' which describes the
+list of initialization operations.
+This file must be stored in the model directory. The model directory structure
+should look like below:
+
+```
+|-- 1
+|   `-- model.graphdef
+|-- config.pbtxt
+`-- init_ops.json
```

Comment:
I was wondering if it would be better to have `init_ops.json` per version of the model? 
A user might want to initialize the variables using different ops for different model versions.

Reply:
How about having a hierarchy? If there is a `init_ops.json` inside the version directory it will be used. Otherwise it will lookup the model folder?

Reply:
Sounds good to me.